eduvpn/ui/stats.py

```diff
@@ -0,0 +1,138 @@
+import os
+import logging
+import time
+from datetime import timedelta
+
+from ..nm import get_iface, get_ipv4, get_ipv6, get_timestamp
+
+logger = logging.getLogger(__name__)
+
+LINUX_STATS_FILE = "/sys/class/net/{iface}/statistics/{filename}"
+
+
+class NetworkStats():
```

Comment:
It would be great to have a test of this, with `LINUX_STATS_FILE` mocked and checking that it parses correctly.

Reply:
great idea!